cypress/locators/show-edit-pod/index.js

```diff
@@ -1,22 +1,36 @@
 import {
-  SHOW_EDIT_POD_TITLE,
-  SHOW_EDIT_SAVE_BUTTON,
-  SHOW_EDIT_CANCEL_BUTTON,
-  SHOW_EDIT_DELETE_BUTTON,
-  SHOW_EDIT_POD_EDIT,
-  SHOW_EDIT_POD_CONTENT_BOX,
+  SHOW_EDIT_POD,
```

Comment:
Why have you changed the format of the naming convention from COMPONENT_ELEMENT to ELEMENT_COMPONENT?